### DISCUSSION TITLE
Railway template? I wish have a way to deploy on railway.app

### DISCUSSION THREAD

**Discussion by zigang93** [Dokploy/dokploy#479] [General]
at least had some manual setup docs?

**Comment by Siumauricio** [comment]
I don't think you can deploy dokploy on railway, basically dokploy raises 2 services once the server starts running, I think railroad would have a limitation for this, besides it uses a file system with a mount, I don't think it's possible either.

If you are interested to deploy a dokploy instance quickly you can see the hostinger service, they offer a image ready to go 

https://support.hostinger.com/en/articles/9822596-how-to-use-the-dokploy-vps-template
[VPS Refferal Link](https://www.hostinger.com/vps-hosting?REFERRALCODE=1SIUMAURICI97)
